core/templates/pages/exploration-player-page/suggestion-modal-for-learner-local-view/suggestion-modal-for-exploration-player.service.ts

```diff
@@ -104,18 +105,7 @@ angular.module('oppia').factory('SuggestionModalForExplorationPlayerService', [
             'learner-suggestion-submitted-modal.template.html'),
           backdrop: true,
           resolve: {},
-          controller: [
-            '$scope', '$uibModalInstance',
-            function($scope, $uibModalInstance) {
-              $scope.close = function() {
-                $uibModalInstance.dismiss();
-              };
-            }
-          ]
-        }).result.then(function() {}, function() {
-          // Note to developers:
-          // This callback is triggered when the Cancel button is clicked.
-          // No further action is needed.
+          controller: 'ConfirmOrCancelModalController'
```

Comment:
I see a modal around line 33, do we need to do any changes there?

Reply:
I don't think so but I think it would be a good idea to test this file's modals.